### PR TITLE
Workspace guide

### PR DESCRIPTION
**Pull request by AlexicaWright**

### CODE REVIEW

File: documentation/pole-workspace-guide.adoc [hunk 1]
```diff
@@ -0,0 +1,419 @@
...
+WITH point(l) AS corrie
+MATCH (x:Location)<-[:OCCURRED_AT]-(c:Crime)
+WITH x, c, point.distance(point(x), corrie) AS distance
+WHERE distance < 500
+RETURN x.address AS address, count(c) AS crime_total, collect(c.type) AS crime_type, distance
+ORDER BY distance
+LIMIT 10
+----
+
+This is a complex query that pipelines the results from one part of the query to the next.
```
Comment (martinohanlon) [inline documentation/pole-workspace-guide.adoc:171]: You could break this query down into a number of steps and ask the learner to run it piece by piece.

e.g. 

1. Find the location and return it so the learner can see what the starting point is
2. Find locations where crimes occured within a distance and see the complete return
3. Order the results and limit them

File: documentation/pole-workspace-guide.adoc [hunk 2]
```diff
@@ -0,0 +1,419 @@
...
+== Distance
+
+Instead of multiple addresses, you can find out how much and what types of crime were committed in the vicinity of one address.
+You can pick any address as your starting point, but for this query you will use an address that may sound familiar.
+
+.Crimes committed in the vicinity of Coronation Street
+[source,cypher]
+----
+MATCH (l:Location {address: '1 Coronation Street'})
+WITH point(l) AS corrie
```
Comment (martinohanlon) [inline documentation/pole-workspace-guide.adoc:162]: Do you need to use `point(l)`? As you have already created a point datatype could you use 

```
WITH l.position AS corrie
```

File: documentation/pole-workspace-guide.adoc [hunk 3]
```diff
@@ -0,0 +1,419 @@
...
+.Reveal the solution
+====
+[source,cypher]
+----
+MATCH (c:Crime {last_outcome: 'Under investigation'})-[i:INVESTIGATED_BY]->(o:Officer {surname: 'Larive'})
+RETURN c.type AS type, count(c) AS number
+ORDER BY count(c) DESC
+----
+
+This should tell you that Officer Larive is currently three Drug crimes
```
Comment (martinohanlon) [inline documentation/pole-workspace-guide.adoc:239]: is currently investigating three Drug crimes

File: documentation/pole-workspace-guide.adoc [hunk 4]
```diff
@@ -0,0 +1,419 @@
...
+
+If you use this information, you can run the following query to bring back only cases under investigation on the drug category.
+
+[source,cypher]
+----
+MATCH (c:Crime {type:'Drugs', last_outcome: 'Under investigation'})-[i:INVESTIGATED_BY]->(o:Officer {surname: 'Larive'})
+return *
+----
+
+You can click on the crime nodes in the graph to expand them and see if any of them share anything else.
```
Comment (martinohanlon) [inline documentation/pole-workspace-guide.adoc:250]: I think you have to double-click on nodes in workspace.

**Review by martinohanlon — COMMENTED** [review]
Looks great. I really like the exploration and use of hints. I have made a few comments relating to language and giving the learner some hints and help.

File: documentation/pole-workspace-guide.adoc [hunk 5]
```diff
@@ -0,0 +1,419 @@
+== Explore a POLE dataset
+:images: {img}
+
+A **P**ersons **O**bjects **L**ocations **Events** datamodel focuses on the relationships between people, objects, locations and events and is a model ideal to be used in law enforcement and intelligence investigations.
+
+image::{img}/model.jpeg[float=right,width=200px]
+
+In this guide, you will learn:
+
```
[the remark this answers is not in the document]
Comment (AlexicaWright) [reply documentation/pole-workspace-guide.adoc:9]: The modeling is already done?

File: documentation/pole-workspace-guide.adoc [hunk 6]
```diff
@@ -0,0 +1,419 @@
...
+[role=NX_TAB_NAV,tab=import]
+pagelaunch::[]
+
+Use the button to import the data into Neo4j.
+
+button::Import POLE[role=NX_IMPORT_LOAD,endpoint=https://neo4j-graph-examples.github.io/pole/data/pole-data-importer.zip]
+
+Crime data for this demo was downloaded from public sources (http://data.gov.uk), and is freely provided for download with locations defined to the block or street level and crimes defined by month only (i.e. no day or timestamp).
+This public crime data does not include any sort of information about persons related to crimes, not even as anonymised tokens - it supplies only crime and location data, or in other words only the 'L' and 'E' for the POLE model.
```
[the remark this answers is not in the document]
Comment (AlexicaWright) [reply documentation/pole-workspace-guide.adoc:29]: You're right, I took this as a chunk from the original guide. I'll rewrite it.
It does include objects though.

File: documentation/pole-workspace-guide.adoc [hunk 7]
```diff
@@ -0,0 +1,419 @@
...
+----
+
+[NOTE]
+====
+The arrow button icon:ArrowIcon[] copies the query to the clipboard.
+The play button icon:PlayIcon[] executes the query and returns the results.
+====
+
+You can see that there are 11 different node labels and that these are connected to each other and themselves by various different relationship types.
+
```
[the remark this answers is not in the document]
Comment (AlexicaWright) [reply documentation/pole-workspace-guide.adoc:46]: Good idea!

File: documentation/pole-workspace-guide.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (martinohanlon) [reply documentation/pole-workspace-guide.adoc:9]: I would say that even if you dont actually do the modelling, by exploring the data model you learn how crime investigation data can be modelled in a graph

File: documentation/pole-workspace-guide.adoc [hunk 8]
```diff
@@ -0,0 +1,419 @@
...
+The `Person` node is especially interesting since it appears to have multiple relationships to itself.
```
[the remark this answers is not in the document]
Comment (AlexicaWright) [reply documentation/pole-workspace-guide.adoc:47]: Indeed!

File: documentation/pole-workspace-guide.adoc [hunk 9]
```diff
@@ -0,0 +1,419 @@
...
+.Different types of crimes
+[source,cypher]
+----
+MATCH (c:Crime)
+RETURN c.type AS crime_type, count (c) AS total
+ORDER BY count(c) DESC
+----
+
+If you recall the graph model, a crime can involve a person, a vehicle or an object.
+
```
[the remark this answers is not in the document]
Comment (AlexicaWright) [reply documentation/pole-workspace-guide.adoc:83]: At this point, I think the user should run the query if they forgot. Plus, there's no room for an image with all the examples.

File: documentation/pole-workspace-guide.adoc [hunk 10]
```diff
@@ -0,0 +1,419 @@
...
+The following query lets you see which crime(s) involved an object:
```
[the remark this answers is not in the document]
Comment (AlexicaWright) [reply documentation/pole-workspace-guide.adoc:84]: I rewrote the query to show what kinds of objects we're talking about. All drug paraphernalia in this dataset-

File: documentation/pole-workspace-guide.adoc [hunk 11]
```diff
@@ -0,0 +1,419 @@
...
+----
+MATCH (o:Object)-[:INVOLVED_IN]->(c:Crime)
+RETURN c.type AS crime_type, count(c) AS total
+ORDER BY count(c) DESC
+----
+
+[NOTE]
+.Challenge
+====
+Can you rewrite the query to show the crimes that involved a person instead of an object?
```
[the remark this answers is not in the document]
Comment (AlexicaWright) [reply documentation/pole-workspace-guide.adoc:97]: That's better, yes.

File: documentation/pole-workspace-guide.adoc (hunk 2 above)
Comment (AlexicaWright) [reply documentation/pole-workspace-guide.adoc:162]: No, ofc. I've amended.

File: documentation/pole-workspace-guide.adoc (hunk 3 above)
Comment (AlexicaWright) [reply documentation/pole-workspace-guide.adoc:239]: Good eye!

File: documentation/pole-workspace-guide.adoc (hunk 1 above)
Comment (AlexicaWright) [reply documentation/pole-workspace-guide.adoc]: Good idea!

File: documentation/pole-workspace-guide.adoc [hunk 12]
```diff
@@ -0,0 +1,449 @@
...
+WITH l.position AS corrie
+MATCH (x:Location)<-[:OCCURRED_AT]-(c:Crime)
+WITH x, c, point.distance(x.position, corrie) AS distance
+WHERE distance < 500
+RETURN x.address AS address, count(c) AS crime_total, collect(c.type) AS crime_type, distance
+ORDER BY distance
+LIMIT 10
+----
+
+Now you have a neat table of locations, how many and what type of crimes committed, and the distance to 1 Coronation Street.
```
Comment (martinohanlon) [inline documentation/pole-workspace-guide.adoc:207]: Should this be 

Now you have a neat table of locations, you can see how many and what type of crimes were committed, and the distance to 1 Coronation Street.

File: documentation/pole-workspace-guide.adoc [hunk 13]
```diff
@@ -0,0 +1,449 @@
...
+
+[NOTE]
+.Challenge
+====
+Can you find the address farthest away from 1 Coronation St and find out what types of crimes have been committed there?
```
Comment (martinohanlon) [inline documentation/pole-workspace-guide.adoc:212]: I like the challenge :)

It allows learners to use what they have learned, modify their query and make it their own.

**Pull request merged**